robottelo/rhsso_utils.py

```diff
@@ -114,44 +109,44 @@ def create_new_rhsso_user(client_id, username=None):
         username = gen_string('alphanumeric')
     RHSSO_NEW_USER['username'] = username
     RHSSO_NEW_USER['email'] = random.choice(valid_emails_list())
-    RHSSO_RESET_PASSWORD['value'] = rhsso_password
+    RHSSO_RESET_PASSWORD['value'] = settings.rhsso.rhsso_password
     upload_rhsso_entity(RHSSO_NEW_USER, "create_user")
     upload_rhsso_entity(RHSSO_RESET_PASSWORD, "reset_password")
     run_command(
-        cmd=f"{KEY_CLOAK_CLI} create users -r {realm} -f create_user",
-        hostname=rhsso_host,
+        cmd=f"{KEY_CLOAK_CLI} create users -r {settings.rhsso.realm} -f create_user",
+        hostname=str(settings.rhsso.host_name),
     )
     user_details = get_rhsso_user_details(RHSSO_NEW_USER['username'])
     run_command(
         cmd="{} update -r {} users/{}/reset-password -f {}".format(
-            KEY_CLOAK_CLI, realm, user_details['id'], "reset_password"
+            KEY_CLOAK_CLI, settings.rhsso.realm, user_details['id'], "reset_password"
         ),
-        hostname=rhsso_host,
+        hostname=str(settings.rhsso.host_name),
     )
     return RHSSO_NEW_USER
 
 
 def update_rhsso_user(username, group_name=None):
     user_details = get_rhsso_user_details(username)
-    RHSSO_USER_UPDATE["realm"] = f"{realm}"
+    RHSSO_USER_UPDATE["settings.rhsso.realm"] = f"{settings.rhsso.realm}"
```

Comment:
The string use of `"realm"` for the dict key shouldn't be changed.

Also, single quotes.

Reply:
Added in latest commit